src/main/java/com/epam/training/auction_backend/controllers/AuctionService.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.training.auction_backend.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.log4j.Logger;
+import org.hibernate.Query;
+import org.hibernate.Session;
+
+import com.epam.training.auction_backend.entity.Auction;
+
+public class AuctionService {
+	
+	public static final Logger logger = Logger.getLogger(AuctionService.class);
```

Comment:
The logger should not be accessed from outside the class, so it should be private.